srcpkgs/mesa/template

```diff
@@ -23,10 +24,11 @@ license="MIT, LGPL-2.1-or-later"
 homepage="https://www.mesa3d.org/"
 changelog="https://docs.mesa3d.org/relnotes/${version}.html"
 distfiles="https://mesa.freedesktop.org/archive/mesa-${version}.tar.xz"
-checksum=ae1b240e11531df528d14dc214d2dc4d2b4f2e835c6230ba0b492b171eceb82b
+checksum=af751b49bb2ab0264d58c31e73d869e80333de02b2d1becc93f1b28c67aa780f
+provides="mesa-etnaviv-dri>=0 mesa-freedreno-dri>=0 mesa-kmsro-dri>=0
```

Comment:
I think this should be `replaces=` and should be in the `mesa-dri` subpkg

Reply:
q66 asked not to remove the sub packages, so I will just revert this part.

Reply:
ok